code/drasil-gool/lib/Drasil/GOOL/LanguageRenderer/Constructors.hs

```diff
@@ -36,11 +36,11 @@ mkStmtNoEnd = flip stmtFromData Empty
 
 -- | Constructs a value in a stateful context
 mkStateVal :: (CommonRenderSym r) => VSType r -> Doc -> SValue r
-mkStateVal = valFromData Nothing Nothing
+mkStateVal = valFromData Nothing (LitInt 0)
```

Comment:
This is suspicious to me: why is the static value defaulting to 0? (same in other places in this PR)